VaticanBAV/et/BAVet117.xml

```diff
@@ -128,32 +227,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <explicit xml:lang="gez">
                               ወተመጠወት፡ ነፍሶ፡ <choice><sic>እግዚእትነ፡</sic></choice> ወአንበረቶ፡ ውስተ፡ ትፍሥሕት፡ እለ፡ ቀሠፍዎ፡ ነስሑ፡ <choice><sic>ወአስቆቀው፡</sic></choice> ኀበ፡ እግዝእትነ፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ጸሎታ፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/>
                             </explicit>
-                            <listBibl type="editions">
```

Comment:
it is correct to remove the editions from here, just make sure that the bibl is there in LIT3612, and elsewhere, thank you